src/options.cpp

```diff
@@ -109,6 +109,31 @@ options_manager::cOpt::cOpt()
     hide = COPT_NO_HIDE;
 }
 
+//add external option
```

Comment:
This comment just repats the function name, what information does it add? Also: chnage the parameters to be references to const strings, no need to force the caller to copy them.